scripts/reanimated_pods.rb

```diff
@@ -0,0 +1,71 @@
+def try_to_parse_rn_package_json(node_modules_dir)
+  begin
+    return JSON.parse(
+      File.read(
+        File.join(node_modules_dir, 'react-native/package.json')
+      )
+    )
+  rescue
+    return nil
+  end
+end
+
+def find_config()
+  result = {
+    :is_user_app => true,
+    :react_native_version => nil,
+    :react_native_minor_version => nil,
+    :is_tv_os_target => false,
+    :react_native_node_modules_dir => nil,
+    :reanimated_node_modules_dir => nil,
+    :react_native_common_dir => nil,
+  }
+
+  rn_node_modules_dir = File.join(File.dirname(`cd #{Pod::Config.instance.installation_root.to_s} && node --print "require.resolve('react-native/package.json')"`), '..')
+  rn_json = try_to_parse_rn_package_json(rn_node_modules_dir)
+
+  if rn_json == nil
+    # user configuration, just in case
+    node_modules_dir = ENV["RN_LOCATION"]
```

Comment:
Does Expo use `RN_LOCATION`? If not, can we rename it to `REACT_NATIVE_NODE_MODULES_DIR`?